dist/tools/compile_commands/compile_commands.py

```diff
@@ -230,6 +251,8 @@ def generate_module_compile_commands(path, state, args):
 
     if args.clangd:
         cdetails.cflags.append('-Wno-unknown-warning-option')
+        # avoid including host system headers
+        cdetails.cflags.append('-nostdinc')
```

Comment:
This line seems to break rust compilation. `-nostdinc` also gets added to the compile commands that the c2rust machinery is using.